roles/confluent.common/tasks/debian.yml

```
@@ -111,7 +111,7 @@
   delay: 5
   when:
     - install_java|bool
-    - custom_apt_repo|bool
+    - repository_configuration == 'custom'
```

think this should be, no matter what java needs to be installed...
```suggestion
    - repository_configuration in ['custom','none']
```